applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h

```diff
@@ -177,7 +177,7 @@ namespace Kratos
 							if (Length < mElemSize) mElemSize = Length;
 						}
 					mElemSize = sqrt(mElemSize);
-					ielem->GetValue(MEAN_SIZE) = mElemSize;
+					ielem->SetValue(MEAN_SIZE, mElemSize);
```

Comment:
why do you change this?

Reply:
To avoid a segmantation fault. GetValue crashes in debug when the variable isn't initialized.

Reply:
The proper to do this would be to firstly allocate the variable by doing a `SetValue` in a threadsafe region. Then you can use the `GetValue` as was done before.